src/npm.ts

```diff
@@ -221,15 +221,18 @@ async function resolveNpmVersion(root: string, {name, range}: NpmSpecifier): Pro
   const cache = await getNpmVersionCache(root);
   const versions = cache.get(name);
   if (versions) for (const version of versions) if (!range || satisfies(version, range)) return version;
-  const href = `https://data.jsdelivr.com/v1/packages/npm/${name}/resolved${range ? `?specifier=${range}` : ""}`;
+  if (range === undefined) range = "latest";
+  const disttag = validRange(range) ? null : range;
+  const href = `https://registry.npmjs.org/${name}${disttag ? `/${disttag}` : ""}`;
   let promise = npmVersionRequests.get(href);
   if (promise) return promise; // coalesce concurrent requests
   promise = (async function () {
     const input = formatNpmSpecifier({name, range});
     process.stdout.write(`npm:${input} ${faint("→")} `);
-    const response = await fetch(href);
+    const response = await fetch(href, {...(!disttag && {headers: {Accept: "application/vnd.npm.install-v1+json"}})});
```

Comment:
This is the “abbreviated” response described here: 

https://github.com/npm/registry/blob/main/docs/responses/package-metadata.md